homeassistant/components/yamaha_musiccast/select.py

```diff
@@ -46,7 +47,7 @@ async def async_select_option(self, option: str) -> None:
         value = {val: key for key, val in self.capability.options.items()}[option]
         await self.capability.set(value)
 
-    @property
+    @cached_property
     def translation_key(self) -> str | None:
         """Return the translation key to translate the entity's states."""
```

Comment:
It looks like this can be set in the constructor in a separate PR

Reply:
#100273